includes/data.php

```diff
@@ -1950,18 +1950,35 @@ function pods_serial_comma( $value, $field = null, $fields = null, $and = null,
 			$format = pods_v( 'repeatable_format', $params->field, 'default', true );
 
 			if ( 'default' !== $format ) {
-				$params->serial = false;
 
-				if ( 'custom' === $format ) {
-					$separator = pods_v( 'repeatable_format_separator', $params->field );
+				switch ( $format ) {
+					case 'ul':
+					case 'ol':
+						if ( is_array( $value ) ) {
```

Comment:
If $value is not an array, should we just force it into an array and do the list HTML anyways?

Reply:
Good one, I think that would indeed be better in this case.